scripts/create-missing-test-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2043
+for folder in tests; do
+	echo "./$folder"
+
+	FILE_NAMES=$(find ./$folder -name "*.go" | grep -v "_test.go" | grep -v "_moq.go" | grep -v "ginkgo" | sed 's/.go//g')
```

Comment:
I would use lowercase for local variable names.

Reply:
Done.